cloud_archives/nwp/icon/icon_eu.py

```diff
@@ -0,0 +1,54 @@
+"""Pipeline for the processing of eu ICON data."""
+import os
+
+import dagster as dg
+
+from cloud_archives.ops.huggingface import (
+    HFFileConfig,
+)
+from cloud_archives.ops.kbatch import (
+    NWPConsumerConfig,
+)
+
+from ._ops import (
+    AssetMaterializationConfig,
+    create_kbatch_huggingface_graph_config,
+    kbatch_huggingface_graph,
+)
+
+# Define the ICON europe zarr archive as a source asset
+icon_europe_zarr_archive = dg.SourceAsset(
+    key=["nwp", "icon", "europe", "zarr_archive"],
+    partitions_def=dg.TimeWindowPartitionsDefinition(
+        fmt="%Y-%m-%d|%H:%M",
+        start="2024-01-31|00:00",
+        cron_schedule="0 0 * * *",
```

Comment:
This could probably be every 6 hours? There is a new run every 3 hours, I think, but so far we've only been archiving the longer ones done every 6 hours

Reply:
Updated with [`689896a` (#57)](https://github.com/openclimatefix/dagster-dags/pull/57/commits/689896aa7d394644835f98d3d08887398e739c6f)